src/main/java/com/google/step/servlets/PromoCodeServlet.java

```diff
@@ -29,30 +29,35 @@ public void init() {
         infoScraper = new YouTubeInfoScraper();
     }
     /**
-     * Takes in formInput as a parameter and then checks for a promoCodes based on the channelId.
-     * If the channelId is real and there are codes the json will return a list of promoCodes.
-     * If the channelId is not proper then the json will return an empty list.
+     * Takes in formInput as a parameter and then checks for a promoCodes based on the userInput.
+     * If the userInput is real and there are codes the json will return a list of promoCodes.
+     * If the userInput is not proper then the json will return an empty list.
      * In a case where an exception is thrown the json will return an empty list.
      */
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        String channelId = request.getParameter("formInput");
+        String userInput = request.getParameter("formInput");
```

Comment:
Factor out the magic string `"formInput"` to a package private final variable since it is also used in tests. Also use the `@VisibleForTesting` google annotation.

Reply:
Done in later PR